invoice/invoice.go

```diff
@@ -31,7 +31,7 @@ func CreateInvoice(ctx context.Context, client *model.Odoo, invoice invoice.Invo
 	if partner.Parent.Valid {
 		nameOnInvoice = partner.Parent.Name
 	}
-	name := fmt.Sprintf("%s APPUiO Cloud %s %d", nameOnInvoice, invoice.PeriodStart.Month(), invoice.PeriodStart.Year())
+	name := fmt.Sprintf("%s %s %s %d", nameOnInvoice, opts.invoiceTitle, invoice.PeriodStart.Month(), invoice.PeriodStart.Year())
```

Comment:
Since the title is mandatory, or the line would look weird, i'd either choose a default (`APPUiO`) if the title is unset, or add the title to the method signature `CreateInvoice(ctx context.Context, client *model.Odoo, invoice invoice.Invoice, invTitle string, options ...Option)`.